spec/Random.Spec.savi

```diff
@@ -85,3 +85,25 @@
     total U32 = 0
     count.times -> (total += random.unbiased_u32_less_than(52))
     assert: (total.f32 / count.f32) == 25.514451980590820
+
+  :it "generates UUIDs version 4, variant 1"
+    random = DeterministicRandom.Xoroshiro128.new_128(1, 2)
+    uuid = random.uuid
+
+    parts = uuid.split('-')
+    version = try (parts[2]![0]! | 0)
+    variant = try (parts[3]![0]! | 0)
+
+    assert: parts.size == 5
+    assert: version == '4'
+    assert: (variant == '8' || variant == '9' || variant == 'a' || variant == 'b')
```

Comment:
I actually didn't know about this! I assumed it was pure random data for the entire string in Ruby's `SecureRandom.uuid` but I see now that I was wrong.

Today I Learned.